ptf/SAI_PTF_user-guide.md

```diff
@@ -0,0 +1,184 @@
+# PTF test user guide
+
+This section describes how to run PTF test cases on your device under test (DUT).
+
+  > This guide covers Ubuntu 18.40 distro and above distros. Steps tested only on Ubuntu 18.04.
+
+`<SAI-URL>` in the documents refers to SAI git repository.
+
+  > **NOTE:** Different SAI branch maps to different SAI header, make sure you use the right version of SAI header.
+
+## Build SAI thrift server and python client (DUT side)
+
+This section describes how to build and install SAI thrift server with python client library for PTF test to run.
+
+### Install prerequisites:
+
+Deb dependencies
+
+```bash
+apt update
+apt install git python python-pip wget doxygen graphviz aspell
+apt install libgetopt-long-descriptive-perl libconst-fast-perl \
+libtemplate-perl libnamespace-autoclean-perl libmoose-perl libmoosex-aliases-perl
+pip install ctypesgen
+```
+
+Apache thrift 0.11.0
```

Comment:
https://github.com/Azure/sonic-buildimage/pull/9884

it should be able to build it with thrift 0.14.0

Reply:
@richardyu-ms  yes, the saithrift can be built with thrift 0.14.1 also. Verified already. Once this PR is merged, I will update the doc with thrift 0.14 info included.